Makefile

```diff
@@ -3,7 +3,7 @@ PYTHON = python
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
 	@echo "  fetch         to fetch the data"
-	@echo "  preproces      to run all scripts after anatomy. LFREQ must be set for highpass cutoff"
+	@echo "  preproces       run sensor space preprocessing from import to epochs"
```

Comment:
preproces -> preprocess

also fix indent

Reply:
I tried to rename every instance of preproces in the makefile but that broke the CI. Is there another file I need to edit?